pdr_backend/sim/sim_engine.py

```diff
@@ -203,8 +224,8 @@ def run_one_iter(self, test_i: int, mergedohlcv_df: pl.DataFrame):
         st.aim.update(acc_est, acc_l, acc_u, f1, precision, recall, loss, yerr)
 
         # track predictoor profit
-        tot_stake = others_stake + stake_amt
-        others_stake_correct = others_stake * pdr_ss.others_accuracy
+        tot_stake = self.others_stake + stake_amt
+        others_stake_correct = self.others_stake * self.others_accuracy
```

Comment:
Just found another issue here, the `self.others_accuracy` value when accuracy endpoint data is used is in this format `52.3` which multiplied by total stake gives a higher value than the total stake.
I think what is expected is that the `self.others_accuracy` values is in this format `0.523`, this was the `others_stake_correct` is a lower number than the `self.others_stake`